workspace/Dockerfile

```diff
@@ -1265,6 +1265,20 @@ RUN if [ ${INSTALL_YAML} = true ]; then \
     ln -s /etc/php/${LARADOCK_PHP_VERSION}/mods-available/yaml.ini /etc/php/${LARADOCK_PHP_VERSION}/cli/conf.d/35-yaml.ini \
 ;fi
 
+###########################################################################
+# RDKAFKA:
+###########################################################################
+
+ARG INSTALL_RDKAFKA=false
+
+RUN if [ ${INSTALL_RDKAFKA} = true ]; then \
+    apt-get install -y librdkafka-dev && \
+    pecl install rdkafka && \
```

Comment:
You need to use rdkafka version to avoid problem with accidentally version upgrade.
Rule is simple - you need to control versions of software in container.

Reply:
I follow the pattern used in the code adding kafka to php-fpm container, I don't really wanna make this different in terms of version, so I'd leave it as it is, thanks for the comment though :)

Reply:
Thanks for the PR.  In env-example, there is `PHP_FPM_INSTALL_RDKAFKA=false` I think you need something similar targeted to workspace.  Maybe `WORKSPACE_INSTALL_RDKAFKA=false`?  Then the workspace Dockerfile will need the same `ARG INSTALL_RDKAFKA=false` that the php-fpm Dockerfile has.